### PR TITLE
Spinner sample

### PR DESCRIPTION
**Pull request by Nasdan**

### CODE REVIEW

File: 11 SpinnerAsync/src/api/memberAPI.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+          member.id = gitHubMember.id;
+          member.login = gitHubMember.login;
+          member.avatar_url = gitHubMember.avatar_url;
+
+          return member;
+        });
+
+        deferred.resolve(members);
+    });*/
```
Comment (brauliodiez) [inline 11 SpinnerAsync/src/api/memberAPI.ts]: Remove this commented code

File: 11 SpinnerAsync/src/components/about/aboutPage.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import header from '../common/header'
+import {Link} from 'react-router';
+
+interface Props {
+}
+
+// Nice tsx guide: https://github.com/Microsoft/TypeScript/wiki/JSX
```
Comment (brauliodiez) [inline 11 SpinnerAsync/src/components/about/aboutPage.tsx]: Remove this comment

File: 11 SpinnerAsync/src/components/common/header.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {Link} from 'react-router';
+
+interface Props {
+}
+
+// Nice tsx guide: https://github.com/Microsoft/TypeScript/wiki/JSX
```
Comment (brauliodiez) [inline 11 SpinnerAsync/src/components/common/header.tsx]: Remove this comment

File: 11 SpinnerAsync/src/components/members/membersPage.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+   public render() {
+     if(!this.props.members)
+        return (<div>No data</div>)
+
+
+       return (
+        <div className="row">
+          <h2> Members Page</h2>
+          <Link to="/member">New Member</Link>
+          <table className="table">
```
Comment (brauliodiez) [inline 11 SpinnerAsync/src/components/members/membersPage.tsx]: Break it into two components (the two tables)

File: 11 SpinnerAsync/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        "./src/**/*.ts",        
+        "./typings/tsd.d.ts",
+        "!./node_modules/**/*"
+    ],
+    "files": [
+        "./typings/tsd.d.ts"
+    ],
+    "atom": {
+        "rewriteTsconfig": false
+    }
```
Comment (brauliodiez) [inline 11 SpinnerAsync/tsconfig.json]: Add the settings to make it "code" compatible

File: 11 SpinnerAsync/src/api/memberAPI.ts (hunk 1 above)
Comment (Nasdan) [inline 11 SpinnerAsync/src/api/memberAPI.ts]: Done

File: 11 SpinnerAsync/src/components/about/aboutPage.tsx (hunk 2 above)
Comment (Nasdan) [inline 11 SpinnerAsync/src/components/about/aboutPage.tsx]: Done

File: 11 SpinnerAsync/src/components/common/header.tsx (hunk 3 above)
Comment (Nasdan) [inline 11 SpinnerAsync/src/components/common/header.tsx]: Done

File: 11 SpinnerAsync/src/components/members/membersPage.tsx (hunk 4 above)
Comment (Nasdan) [inline 11 SpinnerAsync/src/components/members/membersPage.tsx]: Done

**Pull request merged**